py2many/rewriters.py

```diff
@@ -192,13 +196,15 @@ def __init__(self, language):
         super().__init__()
 
     def visit_JoinedStr(self, node):
-        new_node = ast.parse('"".join([])').body[0].value
+        new_node = cast(ast.Expr, ast.parse('"".join([])').body[0]).value
```

Comment:
any `ast.parse(...).body[0]` should use `create_ast_node` whenever we're touching those pieces of code, and it looks like this PR is going to touch most of them.

Reply:
how has this been resolved?

Reply:
Missed it. Resolved in 68695c6